OasisDEX/oasis-borrow
# Automation for Aave

# [[FE] - handling of automation v3](https://app.shortcut.com/oazo-apps/story/8896/fe-handling-of-automation-v3)

<please insert a shortcut link above>
  
## Changes 👷‍♀️
- fixed Modal component warning
- added necessary changes regarding new AutomationBotV2 params
  
## How to test 🧪
- enable `AaveProtectionWrite` and `AaveProtection`
- go to your Aave position
- try to protect yourself
- this should also work when opening a new position
